### DISCUSSION TITLE
Script elencare le interfacce hardware in uso su Linux Debian, Raspberry Pi OS (come da guida)

### DISCUSSION THREAD

**Discussion by tanjama80** [indomus/forum#2735] [HUB: Home Assistant]
Salve a tutti sono neofita nel campo di HA.
Sto eseguendo tutte le meravigliose guide per la sua configurazione e per imparare a gestire l'hub. Ho un Rasp 4pi sul quale ho installato HA OS come Pi Imager.
Devo integrare dei dispositivi Vimar Zigbee ed ho acquistato il coordinator Sonoff, come consigliato.
Ho flashato l'RPC firmware e adesso sono bloccata sullo script di controllo per conoscere la porta logica. Eseguo lo script che dovrebbe tornare questo
/dev/bus/usb/001/001 - Linux_4.19.81-v7+_dwc_otg_hcd_DWC_OTG_Controller_3f980000.usb
/dev/bus/usb/001/007 - dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2119893
/dev/ttyACM1 - dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2119893
/dev/bus/usb/001/003 - 0424_2514
/dev/bus/usb/001/006 - 0424_7800
/dev/bus/usb/001/002 - 0424_2514
/dev/ttyACM0 - 0658_0200
/dev/bus/usb/001/004 - 0658_0200
mi torna questo: 
./ports.sh: line 5: udevadm: command not found
./ports.sh: line 7: udevadm: command not found
/dev/ -
ripetute per 4 volte.

Mi sono collegata col Terminal (spero di aver fatto correttamente perchè via putty non riesco ad accedere al rasp con i canonici 'pi' e 'raspberry') e dovrei procedere in tal senso per installare ZigBee2MQTT.

Vi ringrazio per l'aiuto.
Tanja

**Comment by KC-inDomus** [comment]
> Salve a tutti sono neofita nel campo di HA. Sto eseguendo tutte le meravigliose guide per la sua configurazione e per imparare a gestire l'hub. Ho un Rasp 4pi sul quale ho installato HA OS come Pi Imager. Devo integrare dei dispositivi Vimar Zigbee ed ho acquistato il coordinator Sonoff, come consigliato. Ho flashato l'RPC firmware e adesso sono bloccata sullo script di controllo per conoscere la porta logica. Eseguo lo script che dovrebbe tornare questo /dev/bus/usb/001/001 - Linux_4.19.81-v7+_dwc_otg_hcd_DWC_OTG_Controller_3f980000.usb /dev/bus/usb/001/007 - dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2119893 /dev/ttyACM1 - dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2119893 /dev/bus/usb/001/003 - 0424_2514 /dev/bus/usb/001/006 - 0424_7800 /dev/bus/usb/001/002 - 0424_2514 /dev/ttyACM0 - 0658_0200 /dev/bus/usb/001/004 - 0658_0200 mi torna questo: ./ports.sh: line 5: udevadm: command not found ./ports.sh: line 7: udevadm: command not found /dev/ - ripetute per 4 volte.
> 
> Mi sono collegata col Terminal (spero di aver fatto correttamente perchè via putty non riesco ad accedere al rasp con i canonici 'pi' e 'raspberry') e dovrei procedere in tal senso per installare ZigBee2MQTT.
> 
> Vi ringrazio per l'aiuto. Tanja

Ciao, dal tuo log leggo una ConBee II, non un sonoff ZBdongle-E

**Comment by tanjama80** [comment]
ciao, intanto ti ringrazio per la risposta.
Posso assicurarti che il dongle acquistato è il sonoff.
Quello che leggi è il risultato dello script di esempio copiato dalla guida. A me in realtà lo script torna questo
./ports.sh: line 5: udevadm: command not found
./ports.sh: line 7: udevadm: command not found
/dev/ -

E' come se il rasp non riuscisse a vedere il dispositivo. Se vado in impostazioni e dispositivi, lo vedo SONOFF Zigbee 3.0 USB Dongle Plus V2. 

Lo script, a quanto ho capito, serve per avere una stringa di controllo del dispositivo per proseguire nell'installazione del client MQTT.
Io ho come il presentimento che dal terminal non sono nel punto giusto :/ perchè non riesce a risolvere il comando udevadm.

Spero di essere stata abbastanza chiara. Grazie mille.
